cms/toolbar/items.py

```diff
@@ -236,10 +243,10 @@ def get_context(self):
 class Menu(SubMenu):
     sub_level = False
 
-    def get_or_create_menu(self, key, verbose_name, side=LEFT, position=None):
+    def get_or_create_menu(self, key, verbose_name, disabled=False, side=LEFT, position=None):
```

Comment:
I thought it'd best to keep a consistent signature on related methods. I'll add a note in the release notes.